tox.ini

```diff
@@ -0,0 +1,141 @@
+[tox]
+envlist = py{34,35,36,37}-{all,pip,clang,npm,gem,go,perl,php,cabal,java,java7,java8,scala,elm,r,dart,julia,lua,infer,opam,apt,adhoc,disabled,win,swift,mono}-{list,check,collectonly,skip,noskip}-codecov
+minversion = 3.4
+# Doesnt work
```

Comment:
Not worth raising upstream to tox-travis - which isnt too active and unlikely to fix it soon (I'm one of the contributors)  They have related issues unsolved for years

just remove this comment.